src/table/components/head/HeadCellMenu.tsx

```diff
@@ -30,9 +31,13 @@ const MenuItem = ({
   sortDirection: string;
   sortFromMenu(evt: React.MouseEvent, sortOrder: string): void;
   setOpen: any;
+  isInteractionEnabled: boolean;
   isCurrentColumnActive: boolean;
 }) => {
-  const isDisabled = isCurrentColumnActive && sortOrder === (sortDirection === 'asc' ? 'A' : 'D');
+  let isDisabled = false;
+  isDisabled =
+    (!isInteractionEnabled ? !isDisabled : isDisabled) ||
```

Comment:
So this pattern is never a good idea. There is no reason to set in inital state, then change it right after. The double negation here makes it hard to follow, but I think this should be the same
```suggestion
  const isDisabled = !isInteractionEnabled ||
```

Reply:
.. but I want to turn disable to true when `isInteractionEnabled` is false

Reply:
that is exactly what the `!` operator does